history-of-the-quakers/original/31-book-3-chapter-5.adoc

```diff
@@ -519,13 +539,15 @@ be fore they heard the prisoner`'s defence.
 Then silence being proclaimed, the recorder taking a paper in his hand,
 read to the following purpose, viz.
 
-"`The jury for the king do find that John Crook, John Boston, and Isaac Grey,
+[quote]
+____
+The jury for the king do find that John Crook, John Boston, and Isaac Grey,
```

Comment:
this feels a little more like it should be an embedded "legal" document, since it's a contemporary document that is sort of part of the story.  i think [quote]s should in general be more for when gough is quoting another book about what he's talking about, usually written later.